kafka-consumer-java/src/main/java/target/ITarget.java

```diff
@@ -13,7 +13,8 @@
     default String getOriginalTopic(ConsumerRecord<String, String> record) {
         Iterator<Header> headers = record.headers().headers(Config.ORIGINAL_TOPIC).iterator();
         if (headers.hasNext()) {
-            return new String(headers.next().value());
+            var header = headers.next();
+            return header.value() == null ? null : new String(header.value());
```

Comment:
let's extract this to a helper function `safeGetHeaderValue` since it's user more than once
it's safe to use `null` in the other cases instead of `JSONObject.NULL`